libraries/octree/src/OctreeQuery.cpp

```diff
@@ -17,7 +17,18 @@
 #include "OctreeConstants.h"
 #include "OctreeQuery.h"
 
-OctreeQuery::OctreeQuery() {
+const float DEFAULT_FOV = 45.0f; // degrees
+const float DEFAULT_ASPECT_RATIO = 1.0f;
+const float DEFAULT_NEAR_CLIP = 0.1f;
+const float DEFAULT_FAR_CLIP = 3.0f;
+
+OctreeQuery::OctreeQuery() :
+    _cameraFov(DEFAULT_FOV),
+    _cameraAspectRatio(DEFAULT_ASPECT_RATIO),
+    _cameraNearClip(DEFAULT_NEAR_CLIP),
+    _cameraFarClip(DEFAULT_FAR_CLIP),
+    _cameraCenterRadius(DEFAULT_FAR_CLIP)
+{
```

Comment:
I moved these default values initializations to the CPP file so that we don't need to pollute the global namespace.  I'm using more reasonable default values that allow for the OctreeQueryNode's initial ViewFrustum to be fully initialized (avoids NaN in some of the ViewFrustum's internal data members).